README.md

```diff
@@ -1,32 +1,67 @@
-﻿# LoTAS Mod
-### Low Optimization Tool Assisted Speedrun Mod
+﻿# This project is on hiatus!
+We are focussing our attention to [TASmod](https://github.com/MinecraftTAS/TASmod).  
+For more information and plans read this announcement: [Future of Mineraft TAS tools](https://gist.github.com/ScribbleTAS/43b18e8a92a111e92bf4bb0d9cf49e99)
 
-LoTAS is a Mod used to create [Tool Assisted Speedruns](https://en.wikipedia.org/wiki/Tool-assisted_speedrun). In Minecraft TASing you use a mod called Tickrate Changer to slow the game down. The Game runs at 20 ticks per second so 10 would be 50% speed.
+# LoTAS
+Used to create non-playback, Low optimization Tool-Assisted Speedruns (LoTAS).  
+Essentially, you record the game in slow motion, record yourself speedrunning with a recording program such as [OBS](https://obsproject.com),  
```

Comment:
weird sentence, also it's not speedrunning cuz that sounds suspicously like RTA running.
`You essentially play through your route in slow motion while using a recording program such as [OBS] to capture yourself. Later you speed up the video footage and edit your attempts into a final video.`